.github/workflows/mh_build-and-publish-docs.yaml

```diff
@@ -15,8 +16,23 @@ jobs:
       uses: actions/checkout@v2
       with:
         persist-credentials: false
-    - name: Install requirements
+        
+    - name: Install requirements (minus doxygen)
       run: sudo apt-get install make automake libtool libtool-bin autoconf doxygen gfortran g++ texlive texlive-latex-extra texlive-font-utils
```

Comment:
oops. Meant to remove this, but since I'm overwriting it anyway afterwards it should hopefully have done the right thing. But you're right, doxygen shouldn't be installed (while we have the work-around).